internal/cri/server/sandbox_stats_linux.go

```diff
@@ -72,7 +72,7 @@ func (c *criService) podSandboxStats(
 		podSandboxStats.Linux.Memory = memoryStats
 
 		if sandbox.NetNSPath != "" {
-			rxBytes, rxErrors, txBytes, txErrors := getContainerNetIO(ctx, sandbox.NetNSPath)
+			rxBytes, rxErrors, txBytes, txErrors, rxPackets, rxDropped, txPackets, txDropped := getContainerNetIO(ctx, sandbox.NetNSPath)
```

Comment:
This function returns 8 values. Should we introduce a struct instead?

Reply:
since the getContainerNetIO was used in the container_stats also, I didnt want to introduce a change there. Also, in the initial implementation, we had used a struct , ref: https://github.com/containerd/containerd/pull/10691#discussion_r2041369480, but was later removed based on the comments